packages/ag-charts-types/src/chart/cartesianOptions.ts

```diff
@@ -157,13 +159,26 @@ export interface AgGroupedCategoryDepthOptions<TContext = ContextDefault> {
     tick?: AgGroupedCategoryDepthTickOptions;
 }
 
+type AgAxisIntervalPlacement = 'on' | 'between';
+
+export interface AgAxisCategoryIntervalOptions extends AgAxisBaseIntervalOptions {
+    placement?: AgAxisIntervalPlacement;
+}
+
+export interface AgAxisBandTimeIntervalOptions
+    extends AgAxisContinuousIntervalOptions<AgTimeInterval | AgTimeIntervalUnit | number> {
+    placement?: AgAxisIntervalPlacement;
+}
```

Comment:
I'm not sure if `BandTime` is the best name for this. But I needed something for the unit and ordinal time axes. They incorporate continuous axis options and this category axis placement. So much of the options types changes have been to support this "multi-inheritence".

I don't believe any of this is a breaking change as it is all additive, but some things have changed type.